website/pages/en/seminar.js

```diff
@@ -54,11 +54,10 @@ function Seminar(props) {
                 Substrate together.
               </translate></p>
               <section>
-                <span className='block type--fine-print'>
-                  <a href='https://www.crowdcast.io/e/substrate-seminar'><translate>
-                    join live call »
-                  </translate></a>
-                </span>
+                <a className='btn btn-lg primary-color'
+                  href='https://www.crowdcast.io/e/substrate-seminar'>
+                  <translate>Join Live Call »</translate>
```

Comment:
```suggestion
                  <translate>Signup for Upcoming and Review Past Seminars</translate>
```
To make it clear that both options are there

Reply:
Thanks. I updated the wordings to `Live Call / Past Seminars` to get a balance between conveying the meaning and keeping it short.